Program.cs

```diff
@@ -51,6 +52,53 @@
 }
 
 
+var customers = context.Customers.ToList();
+Console.WriteLine("List of all customers  who don't have sales");
+Console.WriteLine("============================================");
+foreach (Customer c in customers)
+{
+    if (c.Sales.IsNullOrEmpty()) {
+        Console.WriteLine(" " + c.FirstName + " " + c.LastName);
+    }
+}
```

Comment:
Hi @pasinduj, is this already finished? This one looks querying all the customers only.